### DISCUSSION TITLE
hugo-eureka had status 65535

### DISCUSSION THREAD

**Discussion by lennyleadership** [wangchucheng/hugo-eureka#85] [Q&A]
Error in blogdown::new_content("D:/A_Blog/projectname/content/docs/foldername/filename.md") : 
  Failed to create the file 'D:/A_Blog/projectname/content/docs/foldername/filename.md'.
In addition: Warning message:
In system2(find_hugo(), ...) :
  running command '"C:/Users/myname/AppData/Roaming/Hugo/0.82.0/hugo.exe" new "D:/A_Blog/projectname/content/docs/foldername/filename.md" --themesDir themes -t hugo-eureka' had status 65535

我在建第二个博客时，沿用了第一个博客使用的主题。我以为会不会我修改主题时不小心误删了文件或修改了某些参数。于是删了GitHub上的repo，删了本地的项目，重新来过。
执行过blogdown::new_site(theme = "wangchucheng/hugo-eureka")后，
马上执行blogdown::new_content("D:/A_Blog/projectname/content/docs/foldername/filename.md")
报同样的错误。

于是我去第一个博客，
执行blogdown::new_content("D:/A_Blog/projectname/content/docs/foldername/filename.md")，没有问题。

除这两个博客之外，我特意把eureka主题做成一个博客，用于观摩，学习。我在这个博客里执行这条命令，也没有问题。

不知道是blogdown出了问题，还是eureka主题出了问题？

**Comment by wangchucheng** [comment]
如果Eureka可以在某一环境下被使用，那应该不是Eureka主题的问题。因为主题只由hugo处理。由于我没有使用过blogdown，因此我也无法判断这是blogdown的问题还是hugo的问题。

祝好！


________________________________
发件人: Warrior for God ***@***.***>
发送时间: Tuesday, April 6, 2021 10:50:40 AM
收件人: wangchucheng/hugo-eureka ***@***.***>
抄送: Subscribed ***@***.***>
主题: [wangchucheng/hugo-eureka] hugo-eureka had status 65535 (#85)


Error in blogdown::new_content("D:/A_Blog/projectname/content/docs/foldername/filename.md") :
Failed to create the file 'D:/A_Blog/projectname/content/docs/foldername/filename.md'.
In addition: Warning message:
In system2(find_hugo(), ...) :
running command '"C:/Users/myname/AppData/Roaming/Hugo/0.82.0/hugo.exe" new "D:/A_Blog/projectname/content/docs/foldername/filename.md" --themesDir themes -t hugo-eureka' had status 65535

我在建第二个博客时，沿用了第一个博客使用的主题。我以为会不会我修改主题时不小心误删了文件或修改了某些参数。于是删了GitHub上的repo，删了本地的项目，重新来过。
执行过blogdown::new_site(theme = "wangchucheng/hugo-eureka")后，
马上执行blogdown::new_content("D:/A_Blog/projectname/content/docs/foldername/filename.md")
报同样的错误。

于是我去第一个博客，
执行blogdown::new_content("D:/A_Blog/projectname/content/docs/foldername/filename.md")，没有问题。

除这两个博客之外，我特意把eureka主题做成一个博客，用于观摩，学习。我在这个博客里执行这条命令，也没有问题。

不知道是blogdown出了问题，还是eureka主题出了问题？

―
